wgpu-types/src/lib.rs

```diff
@@ -2920,6 +2949,14 @@ impl TextureFormat {
         }
     }
 
+    /// Returns the size multiple requirement for a texture using this format.
+    pub fn size_multiple_requirement(&self) -> (u32, u32) {
+        match *self {
+            Self::NV12 => (2, 2),
+            _ => (1, 1),
```

Comment:
I think this should be the block size for non-nv12. A user might rely on this value without knowing about the whole block thing.